packages/schematics/angular/utility/ast-utils_spec.ts

```diff
@@ -72,7 +71,7 @@ describe('ast utils', () => {
   });
 
   it('should add export to module if not indented', () => {
-    moduleContent = tags.stripIndents`${moduleContent}`;
+    moduleContent = moduleContent.replace(/\n\s*/g, '\n');
```

Comment:
A trick I've used for big blocks of text is to not indent the content and use `.trim()` to clean up leading and trailing whitespace like so:

```typescript
describe(() => {
  beforeEach(() => {
    moduleContent = `
// Start text at 0-indent, so no lines have leading spaces.

import { BrowserModule } from '@angular/platform-browser';
import { NgModule } from '@angular/core';
import { AppComponent } from './app.component';

@NgModule({
  declarations: [AppComponent],
  imports: [BrowserModule],
  providers: [],
  bootstrap: [AppComponent]
})
export class AppModule { }
    `.trim(); // Indent aligns with the start of the statement. Cleans up leading and trailing whitespace.
  });
});
```

I find that easier to read and less likely to screw up with a regex replacement.

Reply:
Based on the test description, i was assuming that it was testing the case where there was no indentation at all and specifically within the `NgModule` metadata.
I'll revert these specific test related changes for now since the tests should get a larger overhaul when the APIs under test get cleaned up.